src/commands/lambda/flare.ts

```diff
@@ -86,6 +90,17 @@ export class LambdaFlareCommand extends Command {
       errorMessages.push(commonRenderer.renderError('No email specified. [-e,--email]'))
     }
 
+    // Validate start/end flags if both are specified
+    let startMillis = 0
+    let endMillis = 0
+    try {
+      ;[startMillis, endMillis] = validateStartEndFlags(this.start, this.end)
```

Comment:
If no start/end flags are specified, I set them to 0.
Would it be more readable to leave `startMillis` and `endMillis` as undefined if the start/end flags were not specified?

Reply:
Why `;` at the beginning of the array definition? Is it because you want to assign the value to a previously defined variable? Maybe a comment about this could help to understand why this is necessary.

Reply:
It was required by ESLint, no other reason

Reply:
I think we should change it to `undefined`, since it is more readable, now seeing how I came up with questions below around `rangeSpecified`, it makes sense to leave it to undefined. Although if user specifies 0, we'd have to also change that.

What happens if it's a negative number?